Java/other/BenjaminBulbs.java

```diff
@@ -0,0 +1,25 @@
+/*Given n bulbs. At first, all of them are toggled.
+In the second go, every second bulb is toggled. 
+In the third go, every third bulb is toggled, and so on.
+Find the number of bulbs that will be ON after the process ends.
+*/
+
+import java.util.*;
+
+public class BenjaminBulbs{
+    public static void main(String args[]){
```

Comment:
Shift all the primary Code Logic to another function. `main()` is primarily used for the driver code purpose.